Sources/MlemMiddleware/Content Models/Post/Post2Providing.swift

```diff
@@ -89,6 +97,19 @@ public extension Post2Providing {
         get async { await api.markReadQueue.ids.contains(self.id) }
     }
     
+    @discardableResult
+    func toggleHidden() -> Task<StateUpdateResult, Never> {
+        updateHidden(!hidden)
+    }
+    
+    @discardableResult
+    func updateHidden(_ newValue: Bool) -> Task<StateUpdateResult, Never> {
+        // Unlike other
```

Comment:
?

Reply:
Oops, must have started typing it and then forgot to finish it haha